pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -715,6 +716,20 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get", "put", "update", "delete").URLs(templateapi.ServiceBrokerRoot + "/*").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "resource-metrics-view",
+				Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-view": "true", "rbac.authorization.k8s.io/aggregate-to-admin": "true"}},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule(read...).Groups(metrics.GroupName).Resources("pods").RuleOrDie(),
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "resource-metrics-cluster-view", Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-cluster-reader": "true"}},
```

Comment:
Name does not look correct to me.

Maybe something like:

`system:openshift:aggregate-to-cluster-reader:resource-metrics`

Reply:
Also, should probably go in a const like the other role names